tools/data_converter/main.go

```diff
@@ -75,7 +75,7 @@ func loadConfig(path string) (*bwmf.Config, error) {
 
 // NOTE(baigang): Each line represents an element, formated as "rowID columnID value".
 // The `rowID`s are ASSUMED to be sorted and grouped (as in reducer tasks of a Hadoop job).
-func txtToMatPb(n int) (*pb.MatrixShard, error) {
```

Comment:
Please take a look https://github.com/JuliaLang/julia/blob/master/base/sparse/csparse.jl#L15 .

I've found some issues about this tool. I am using `julia` to convert sparse matrix representation for now.